apps/meteor/client/startup/audit.tsx

```diff
@@ -57,6 +62,17 @@ onToggledFeature('auditing', {
 					</MainLayout>,
 				),
 			},
+			{
+				path: '/security-logs',
+				id: 'security-logs',
+				element: appLayout.wrap(
+					<MainLayout>
+						<PermissionGuard permission='can-audit-log'>
```

Comment:
In the endpoint we use the `can-audit` permission. `can-audit-log` is used for the `Audit Logs` (just `Logs` in the menu), which are the logs of the audit actions. For example, if I audit a room, a log will be created in the `Audit Logs` showing that my user audited a room.